vendor.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'product.rb'
+
+class Vendor
+  attr_reader :name_of_vendor
+
+  def initialize(name, product)
```

Comment:
As far as there can be multiple products, let's make the `product` as a plural => `products`
There is no need to name the variables inside the class like `name_of_vendor`. As far as they are already inside Vendor it is enough to have variables `'name', 'products'`